### PR TITLE
Add german translations

### PR DESCRIPTION
**Pull request by alex289**
Hey, I would like to add german translations to this project.

In german there are two ways to say "you". One is more formal and one is more personal. For financing most platforms choose the formal one so I did too. I can change it to the personal one if you want

### CODE REVIEW

File: config/locales/views/passwords/de.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+de:
+  passwords:
+    edit:
+      password: Neues Passwort
+      password_challenge: Derzeitiges Passwort
+      submit: Password zurücksetzen
```
Comment (lewis-wf) [inline config/locales/views/passwords/de.yml:7]: Password or passwor**t** here?

Comment (alex289) [reply config/locales/views/passwords/de.yml]: Good eye, thank u

**Comment by zachgoll** [comment]
@alex289 really appreciate the contribution but I think at the moment we should hold off on adding new translations.

We are pushing new features on a daily basis that require additional translations, so these will quickly fall out of sync.

Long term we will be prioritizing I18n but due to the global nature of this repo we will likely need to build an automated, AI-assisted process to keep up with the tens of languages we need to translate for.

**Comment by alex289** [comment]
> @alex289 really appreciate the contribution but I think at the moment we should hold off on adding new translations.
> 
> 
> 
> We are pushing new features on a daily basis that require additional translations, so these will quickly fall out of sync.
> 
> 
> 
> Long term we will be prioritizing I18n but due to the global nature of this repo we will likely need to build an automated, AI-assisted process to keep up with the tens of languages we need to translate for.

No problem 😄. If you still need some help with the german translations I would love t help.

**Pull request closed without merging**